src/CropNerGUI.py

```diff
@@ -33,24 +36,40 @@ class CropNerGUI:
     def __init__(self):
         # Create a GUI window
         self.rootWin = tk.Tk()
-        # self.rootWin.option_add('*Font', 'Times 24')
+        #self.rootWin.option_add('*Font', 'Times 24')
         self.rootWin.title("GEMS NER Annotation Tool")
         self.rootWin.geometry('1500x900')
 
-        self.model_dir = None
         self.content=[""]
+
         self.tags=["highlight","default_color_tag","ALAS","CROP","CVAR","JRNL","PATH","PED","PLAN","PPTD","TRAT"]
         self.colors=["gray","black","violet","lawn green","deep sky blue","yellow","red","orange","pink","brown","MediumPurple1"]
+
         self.tag_colors_buttonID = {}
         self.crop_cnt = {}
         self.cvar_cnt = {}
 
-        self.raw_file = None
+        # allows default options for model and file for when the GUI is run, the first arg is the model path and the second arg is the file path
+        if len(sys.argv) >= 2:
+            self.model_dir = sys.argv[1]
+        else:
+            self.model_dir = None
+
+        if len(sys.argv) >= 3:
+            self.raw_file = sys.argv[2]
+            # flag to take care of loading pdfs from a default path which is a tiny bit different from loading a user selected path
+            self.default_file = True
+        else:
+            self.raw_file = None
+            self.default_file = False
+
+
         self.annotation_file = None
         self.sentences = None
         self.annotation_dict = {}
         self.file_extension = None
         self.nlp_agdata = None
+        self.nlp_pos = spacy.load("en_core_web_lg") # spacy model to use for pos
```

Comment:
I am assuming this is a default setting and it gets updated if a user selects their own model?

Reply:
Right now it is hardcoded in for the pos model to use the en_core_web_lg, so I still need to add the functionality to switch.